packages/andromeda-non-fungible-tokens/src/crowdfund.rs

```diff
@@ -40,7 +40,7 @@ pub enum ExecuteMsg {
     /// Purchase tiers with cw20
     Receive(Cw20ReceiveMsg),
     /// End the campaign
-    EndCampaign {},
+    EndCampaign { is_discard: bool },
```

Comment:
I think it might be better to make this a separate message so that it isn't accidentally called by the owner.

Reply:
You could use the same handler but I'd prefer it's an explicit message.

Reply:
Added `DiscardCampaign` message to separate discard/end campaign messages. They are still processed by one handler.